guidance/src/guidance/guidance_worker.cpp

```diff
@@ -180,6 +191,9 @@ namespace guidance
         pnh_.getParam("required_plugins", required_plugins);
         process_required_plugin_list(required_plugins);
 
+        pnh_.getParam("vehicle_type", vehicle_type);
```

Comment:
After moving this parameter to the CARMAConfig you will need to access it using the regular nh_